### ISSUE TITLE
Looks like boolean Defaults cause problems.

### ISSUE THREAD

**Issue by dafydd2277** [jeremypruitt/sudoers-report#1]
```
[ dafydd2277@localhost ] $ sudo ./sudoers-report --filename /etc/sudoers --hostname localhost
UNKNOWN LINE: Defaults    requiretty
UNKNOWN LINE: Defaults    env_reset
Can't use an undefined value as a HASH reference at ./sudoers-report line 192.
```

Here are the Defaults lines from my `/etc/sudoers` file:

```
Defaults    requiretty

Defaults    env_reset
Defaults    env_keep = "COLORS DISPLAY HOSTNAME HISTSIZE INPUTRC KDEDIR \
                        LS_COLORS MAIL PS1 PS2 QTDIR USERNAME \
                        LANG LC_ADDRESS LC_CTYPE LC_COLLATE LC_IDENTIFICATION \
                        LC_MEASUREMENT LC_MESSAGES LC_MONETARY LC_NAME
LC_NUMERIC \
                        LC_PAPER LC_TELEPHONE LC_TIME LC_ALL LANGUAGE LINGUAS \
                        _XKB_CHARSET XAUTHORITY"
```

It looks like it's choking on the Defaults values that aren't equalities.

**Comment by dafydd2277** [comment]
Hmmm.... Nope, something else is going on. I still get the error after commenting out those two entries.

```
[ dafydd2277@localhost ] $ sudo ./sudoers-report --filename /etc/sudoers --hostname localhost
Can't use an undefined value as a HASH reference at ./sudoers-report line 192.
```

I'm looking at it, but Perl isn't a strength of mine.

**Comment by dafydd2277** [comment]
Got it. Line 192:

```
my %host_alias = %{ $sudoers->{'Host_Alias'} };
```

will only work if the `sudoers` file actually has uncommented `Host_Alias` lines. Mine doesn't, as every host gets its own `suders` file. Once I faked up a `Host_Alias` line, the script ran fine. So, maybe the fix is to handle the exception and drop out of `get_host_alias_names_for_hostname()`?

**Comment by jeremypruitt** [comment]
You are absolutely right about that. I just committed a test to cover this case. Now I'll work to make the test pass.

**Comment by jeremypruitt** [comment]
Ok, both the Defaults line warnings and the exception thrown when no Host Alias directives are present (should) have been addressed in the 0.0.4 release I just pushed.
